go.mod

```diff
@@ -3,65 +3,77 @@ module github.com/cert-manager/sample-external-issuer
 go 1.20
 
 require (
-	github.com/cert-manager/cert-manager v1.8.0
-	github.com/go-logr/logr v1.2.0
-	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.23.4
-	k8s.io/apimachinery v0.23.4
-	k8s.io/client-go v0.23.4
-	sigs.k8s.io/controller-runtime v0.11.1
+	github.com/cert-manager/cert-manager v1.11.1
```

Comment:
This is the only line I changed manually.
The rest was changed by `go mod tidy`.